tests/jenkins/TestPromoteArtifacts.groovy

```diff
@@ -55,41 +56,67 @@ class TestPromoteArtifacts extends BuildPipelineTest {
             return [{}]
         })
 
-        helper.addShMock('find workspace/artifacts/vars-build/1.3.0/33/x64/linux/builds/opensearch/core-plugins -type f') { script ->
+        helper.addShMock('find workspace/artifacts/vars-build/1.3.0/33/linux/x64/builds/opensearch/core-plugins -type f') { script ->
             return [stdout: "tar_dummy_artifact_1.0.0.tar.gz zip_dummy_artifact_1.1.0.zip dummy_artifact_1.1.0.dummy", exitValue: 0]
         }
         helper.addShMock('sha512sum tar_dummy_artifact_1.0.0.tar.gz') { script ->
-            return [stdout: "shaHashDummy_tar_dummy_artifact_1.0.0.tar.gz  workspace/artifacts/vars-build/1.3.0/33/x64/linux/builds/opensearch/core-plugins/tar_dummy_artifact_1.0.0.tar.gz", exitValue: 0]
+            return [stdout: "shaHashDummy_tar_dummy_artifact_1.0.0.tar.gz  workspace/artifacts/vars-build/1.3.0/33/linux/x64/builds/opensearch/core-plugins/tar_dummy_artifact_1.0.0.tar.gz", exitValue: 0]
         }
         helper.addShMock('sha512sum zip_dummy_artifact_1.1.0.zip') { script ->
-            return [stdout: "shaHashDummy_zip_dummy_artifact_1.1.0.zip  workspace/artifacts/vars-build/1.3.0/33/x64/linux/builds/opensearch/core-plugins/zip_dummy_artifact_1.1.0.zip", exitValue: 0]
+            return [stdout: "shaHashDummy_zip_dummy_artifact_1.1.0.zip  workspace/artifacts/vars-build/1.3.0/33/linux/x64/builds/opensearch/core-plugins/zip_dummy_artifact_1.1.0.zip", exitValue: 0]
         }
-        helper.addShMock('basename workspace/artifacts/vars-build/1.3.0/33/x64/linux/builds/opensearch/core-plugins/tar_dummy_artifact_1.0.0.tar.gz') { script ->
+        helper.addShMock('basename workspace/artifacts/vars-build/1.3.0/33/linux/x64/builds/opensearch/core-plugins/tar_dummy_artifact_1.0.0.tar.gz') { script ->
             return [stdout: "tar_dummy_artifact_1.0.0.tar.gz", exitValue: 0]
         }
-        helper.addShMock('basename workspace/artifacts/vars-build/1.3.0/33/x64/linux/builds/opensearch/core-plugins/zip_dummy_artifact_1.1.0.zip') { script ->
+        helper.addShMock('basename workspace/artifacts/vars-build/1.3.0/33/linux/x64/builds/opensearch/core-plugins/zip_dummy_artifact_1.1.0.zip') { script ->
             return [stdout: "zip_dummy_artifact_1.1.0.zip", exitValue: 0]
         }
 
-        Path source = Path.of("tests/data/opensearch-build-1.1.0.yml");
-        target = Path.of("artifacts/vars-build/1.3.0/33/x64/linux/builds/opensearch/manifest.yml");
+        targetOpenSearch = copy(
+            "tests/data/opensearch-build-1.1.0.yml", 
+            "artifacts/vars-build/1.3.0/33/linux/x64/builds/opensearch/manifest.yml"
+        );
+
+        targetOpenSearchDashboards = copy(
```

Comment:
I'd prefer we create and commit this folder structure with the files under `tests/data/artifacts/..` instead of always doing it in `setUp`, and having to cleanup afterwards.

Reply:
It needs to update the `WORKSPACE` param and all related test setup (quite a lot). I can create another issue to track this effort.

Reply:
follow up issue created https://github.com/opensearch-project/opensearch-build/issues/1494